packages/opentelemetry-core/src/trace/sampler/ProbabilitySampler.ts

```diff
@@ -23,6 +24,8 @@ export class ProbabilitySampler implements Sampler {
   }
 
   shouldSample(parentContext?: SpanContext) {
+    if (parentContext && parentContext.traceFlags !== undefined)
```

Comment:
Could you please add comment here: "If the parent is sampled keep (or respect) the sampling decision."

Reply:
> Could you please add a comment here: "If the parent is sampled keep (or respect) the sampling decision." or something like this.